apps/client/src/features/editors/Editor.module.scss

```diff
@@ -67,6 +67,14 @@ $playback-width: 26rem;
   }
 }
 
```

Comment:
See the other points from the issue description #477 


- Increate root font size to 17px: this will allow the text to take more space, and potentially improve experience in high-density screens
- Allow rundown column to expand: Currently the rundown is capped at `44.5rem`, a simple change to allow expansion could go a long way. I have tested with `minmax(44.5rem, 1000px)` and it seemed ok, although a real value might take some experimentation
- Allow central column to grow: the central column is capped at `26rem`, I have not experimented but assume this could grow slightly without issues. It would take some experimentation
- Allow rightmost column to grow: similar as the rest, this one is capped at `800px`, It would be interesting to see if we could grow this